eks_fargate/README.md

```diff
@@ -124,6 +124,8 @@ spec:
          value: "datadoghq.com"
        - name: DD_EKS_FARGATE
          value: "true"
+       - name: DD_CLUSTER_NAME
+         value: "<CLUSTER_NAME>"
```

Comment:
I think can be removed, but we can keep the note

Reply:
There is no way to auto-discover the cluster name on EKS Fargate, I think it's important to remind the user to set the cluster name manually via `DD_CLUSTER_NAME`.

Reply:
> the cluster_name can't be provided by the cluster-agent?

No, it's not the case currently

Reply:
Done (CONT-3219)